.travis.yml

```diff
@@ -41,10 +41,15 @@ matrix:
       env: PROJECT_DIR=examples/czmq TOOLCHAIN=clang-libstdcxx
     - os: linux
       env: PROJECT_DIR=examples/czmq TOOLCHAIN=gcc-4-8
-    - os: linux
-      env: PROJECT_DIR=examples/czmq TOOLCHAIN=android-ndk-r10e-api-19-armeabi-v7a-neon
-    - os: linux
-      env: PROJECT_DIR=examples/czmq TOOLCHAIN=analyze
+    
+    # FIXME:
```

Comment:
Please leave some info about problem. At least the link to the broken Travis job.